DigraphyApi/Repository/OrderRepository.cs

```diff
@@ -0,0 +1,19 @@
+using DigraphyApi.Data;
+using DigraphyApi.Interfaces;
+using DigraphyApi.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace DigraphyApi.Repository;
+
+public class OrderRepository(AppDbContext context) : IOrderRepository
+{
+    public async Task<ICollection<Order>> GetOrdersAsync(bool verified)
+    {
+        return await context.Orders.Where(o => !o.IsVerified && verified == false).OrderBy(o => o.Id).ToListAsync();
+    }
+
+    public async Task<ICollection<Order>> GetOrdersByCollectionIdAsync( int collectionId, bool verified)
+    {
+        return await context.Orders.Where(o => !o.IsVerified && verified == false && o.Collection.Id  == collectionId).OrderBy(o => o.Id).ToListAsync();
+    }
```

Comment:
Rather than having two separate methods, I would combine it into one and make it take a `OrderFilters` object as a parameter. Then you can conditionally build the query like so:

```cs
var query = context.Orders;

if (filters.collectionId != null) {
    query = query.Where(...);
}
if (filters.verified != null) {
    query = query.Where(...);
}
```

It's perfectly fine to have multiple `Where`s. They get automatically ANDed together I'm pretty sure. This also makes it easier to add other query parameters for filtering later if we want.